notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,925 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    DESIGN_LAYOUT = {'Info_Probe':[14, 36, 410, 135],  # feature_name, xmin, ymin, xmax, ymax
</pre><p>I believe we should put <code>DESIGN_LAYOUT </code> config after the visualization of Transform result, but not in advance. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>